### ISSUE TITLE
Filters selection countering issue

### ISSUE THREAD

**Issue by marko-srb** [WordPress/wporg-showcase-2022#247]
We have a logic issue in the preivew/counters. 

Under any filter, we have counters for two different logics:

1. For each individual tag (Art), or category (Business) we have a counter of 'how many sites' there are under this selected option
2. For each filter, we have a counter of how many individual tags are applied. 

These are two different logics and are confusing. I know we had feedbacks like the following:

a. We need counter of the choices selected for each filter, next to Apply, due accessibility reasons.
b. People then think those represent number of websites. 

In my attempt to solve the issue, I tried putting a black box behind the number, which indicates the same visual element as the one that previews number of selected choices once the filter modal is closed. I am not happiest with this solution, but it seems the only one that is working better than what we have (suggestion 2 from the screenshot). If you have any other ideas, please put them here. Otherwise, I maybe would suggest we remove the website counters all together... 

I also tried to find similar behavior among complex filtering mechanism on various sites and couldn't find one where both logics are present at the same time. 

![filtering-numbers-issue](https://github.com/WordPress/wporg-showcase-2022/assets/111751264/b1ea150d-1ba7-4d0e-8e3d-615b8b2cae43)

CC: @jasmussen , @fcoveram , @ndiego

**Comment by jasmussen** [comment]
I agree, the logic is confusing. The numbers are confusing not just because they denote different counts (tags vs. filters applied), but also because despite those numbers you can very easily end up in a situation where you have zero results, and there's no number shown for that. 

To that end, I don't see the value of having any number at all. I do think that the number on the button itself is meant to indicate that you've selected more than one filter. So if we have to keep that, I'd remove the count next to each tag.

**Comment by fcoveram** [comment]
I agree with @jasmussen. The confusion of mixing two logics is bigger than the need to show how many sites users will see. I'm drawn to remove the number in each tag and keep the filters selected in the button.

**Comment by ndiego** [comment]
> I agree with @jasmussen. The confusion of mixing two logics is bigger than the need to show how many sites users will see. I'm drawn to remove the number in each tag and keep the filters selected in the button.

Fine with me, but I just want to note that this brings us back to the prior implementation that generated the original user feedback that it was confusing. So we should still keep this issue, or create another, for further explorations in the future.

**Comment by ndiego** [comment]
Let's move forward with reverting the indicators for how many sites in each taxonomy term. We can revisit post-launch. Thanks for all your work on this @renintw 🙏
